source/daplink/drag-n-drop/flash_manager.c

```diff
@@ -48,7 +48,7 @@ __attribute__((aligned(4)))
 static uint8_t buf[1024];
 static bool buf_empty;
 static bool current_sector_valid;
-static bool page_erase_enabled = false;
+static bool page_erase_enabled = true;
```

Comment:
I'm slightly concerned about making this change globally, only because most of the flash algos have not been tested in page erase mode. You can use set the `kEnablePageErase` flag in the `board_info_t` struct to set it per board. It's _probably_ safe to change it globally, but that would better be done in a separate PR.

Reply:
added kEnablePageErase to source/board/mimxrt1170_evk.c.